src/ray/gcs/gcs_server/gcs_server.cc

```diff
@@ -143,36 +147,52 @@ std::unique_ptr<rpc::ObjectInfoHandler> GcsServer::InitObjectInfoHandler() {
 }
 
 void GcsServer::StoreGcsServerAddressInRedis() {
-  boost::asio::ip::detail::endpoint primary_endpoint;
-  boost::asio::ip::tcp::resolver resolver(main_service_);
-  boost::asio::ip::tcp::resolver::query query(
-      boost::asio::ip::host_name(), "",
-      boost::asio::ip::resolver_query_base::flags::v4_mapped);
+  boost::asio::io_service io_service;
+  boost::asio::ip::tcp::socket socket(io_service);
   boost::system::error_code error_code;
-  boost::asio::ip::tcp::resolver::iterator iter = resolver.resolve(query, error_code);
-  boost::asio::ip::tcp::resolver::iterator end;  // End marker.
+  socket.connect(boost::asio::ip::tcp::endpoint(
+                     boost::asio::ip::address::from_string("8.8.8.8"), 53),
```

Comment:
Can we put `8.8.8.8` in `ray/common/constants.h` now that it's used both here and in the raylet?

Reply:
fixed, thanks